src/components/NavbarMain/NavbarMain.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './NavbarMain.scss';
+
+const NavbarMain = () => {
+  return (
+    <nav className="navbar">
```

Comment:
최상위 클래스네임은 컴포넌트 이름이랑 동일할게 하는 컨벤션으로 맞춰주세요!
```suggestion
    <nav className="navbarMain">
```